models/enterprise_sonic/static_routes/merged_example_01.txt

```diff
@@ -0,0 +1,45 @@
+# Using merged
```

Comment:
It would be helpful to add a "blackhole" next-hop example, and a merged/modify case that changes an attribute.